### DISCUSSION TITLE
TumblThree won't save metadata as json when I set it to that, what happened?

### DISCUSSION THREAD

**Discussion by honeylemontea** [TumblThreeApp/TumblThree#444] [Q&A (Question & Answer)] [answered]
I have TumblThree ver. 2.10.399 running on Windows 10 Desktop. The problem I'm having is that I have set metadata to save all the blogs I intended to download to json, but when I go in to view the files what I see is that all the text stuff saves as plain text instead.

Did I do something wrong? How do I fix it? Any help will be appreciated. Thanks in advance!

**Comment by thomas694** [comment, marked as answer]
Did you add your blogs before changing to "json" in the settings dialog? Then in the blog details it still says "text", because the settings dialog is only valid for newly added blogs. Settings of existing blogs need to be changed on the blog's detail page.

Otherwise please let us know more details. What kind of blogs (column "Type") are you trying to download?
